src/theme/basic/templates/body-cell-array-of-number.tpl.html

```diff
@@ -1,6 +1,6 @@
 <ng-template key='body-cell-array-of-number.tpl.html'
 			 let-$cell>
-	<ul>
+	<ul class="q-grid-array">
```

Comment:
remove q-grid-array class it's not necessary

Reply:
done in the last commit